x-pack/plugins/triggers_actions_ui/public/plugin.ts

```diff
@@ -22,15 +22,23 @@ export interface TriggersAndActionsUIPublicPluginSetup {
   alertTypeRegistry: TypeRegistry<AlertTypeModel>;
 }
 
-export type Start = void;
+export interface TriggersAndActionsUIPublicPluginStart {
+  actionTypeRegistry: TypeRegistry<ActionTypeModel>;
+  alertTypeRegistry: TypeRegistry<AlertTypeModel>;
+}
 
 interface PluginsStart {
   data: DataPublicPluginStart;
   charts: ChartsPluginStart;
   management: ManagementStart;
 }
 
-export class Plugin implements CorePlugin<TriggersAndActionsUIPublicPluginSetup, Start> {
+export class Plugin
+  implements
+    CorePlugin<
+      TriggersAndActionsUIPublicPluginSetup,
+      TriggersAndActionsUIPublicPluginStart | undefined
```

Comment:
Same question from @pmuellr in another PR -> https://github.com/elastic/kibana/pull/58514#discussion_r388271463.